recipes/seismon/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "seismon" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/seismon-{{ version }}.tar.gz
+  sha256: ceff57b53e3463a84408425f0ad3feba2c5eb181bef3176af41e5aca15274a23
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - arrow
+    - flask-caching
+    - flask-login
+    - flask-sqlalchemy
+    - flask-wtf
+    - gwpy
+    - lxml
+    - matplotlib-base >=2.2.0
+    - numpy >=1.7.1
+    - obspy
+    - passlib
+    - python
+    - redis-py
+    - scipy >=0.12.1
+    - simplejson
+
+test:
+  imports:
+    - seismon
+    - utils
```

Comment:
This should be removed, anticipating a future release that includes https://github.com/gwdetchar/seismon/pull/5.

```suggestion
```